src/adapter.rs

```diff
@@ -109,7 +117,9 @@ pub trait BluetoothAdapter {
     fn set_can_stop_discovery(&self, can_stop_discovery: bool) -> Result<(), Box<Error>>{
         Err(Box::from(NOT_SUPPORTED_ERROR))  
     }
-    fn create_discovery_session(&self)-> Result<BluetoothDiscoverySession, Box<Error>> ;
+    fn create_discovery_session(&self)-> Result<BluetoothDiscoverySession, Box<Error>> {
```

Comment:
This does not seem like a change we want to make. All adapter implementations should be providing an implementation of this method instead.

Reply:
This was added when we were facing issues while trying to implement trait for EmptyAdapter in empty.rs. I should have removed it. Sorry